Makefile

```diff
@@ -89,7 +89,7 @@ test/unit:
 .PHONY: test/e2e
 test/e2e: export GH_CLIENT_ID := 1234
 test/e2e: export GH_CLIENT_SECRET := 1234
-test/e2e: cluster/cleanup cluster/prepare cluster/prepare/configmaps
+test/e2e: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/configmaps cluster/prepare/crd deploy/integreatly-installation-cr.yml
```

Comment:
How about have `cluster/cleanup` depend on `cluster/cleanup/crds`, and `cluster/prepare` depend on `cluster/prepare/configmaps` and `cluster/prepare/crd`?

I'm not sure what expectations you folks have around running `cluster/prepare` or `cluster/cleanup, so it may not be appropriate.

Reply:
I'm good with having `cluster/cleanup` to depend on `cluster/cleanup/crds` as it looks like the `cluster/cleanup` is only used from the `test/e2e` command

Also good with `cluster/prepare` to depend on `cluster/prepare/configmaps` and `cluster/prepare/crd` for the same reason.

From what I can see `cluster/prepare` and `cluster/cleanup` both are just used in `test/e2e` though am not sure are people using them much outside of this